.github/workflows/ci.yml

```diff
@@ -19,9 +16,14 @@ jobs:
     runs-on: depot-ubuntu-22.04-arm
     outputs:
       buildId: ${{ steps.build.outputs.build-id}}
+    permissions:
+      id-token: write
+      contents: read
```

Comment:
Nitpick: `zizmor` doesn't currently suggest this but I'd recommend having a comment next to `id-token: write`, so that future reviewers can figure out why the job needs an OIDC credential (since they're pretty powerful).

OTOH this already didn't have one so I think it's OK to punt on this for a follow-up 🙂